lib/rubygems/version.rb

```diff
@@ -279,7 +279,10 @@ def encode_with coder # :nodoc:
   # A version is considered a prerelease if it contains a letter.
 
   def prerelease?
-    @prerelease ||= !!(@version =~ /[a-zA-Z]/)
+    unless defined?(@prerelease)
```

Comment:
@segiddins I think it's silly to replace a perfectly fine Ruby operator with a method call, but a quick benchmark shows it's only 13% slower so I'll change it.